### PR TITLE
User can review order

### PR DESCRIPTION
**Pull request by erikwesterberg**
- Added feature test
- Generate Cartify views
- Add render Object-list to 1st checkout step
- Refactor and delete unnecessary and out of scope views
- Refactor feature test to fit with new feature
- Pivotalt - https://www.pivotaltracker.com/n/projects/2360244/stories/167007918

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -27,4 +27,9 @@
   within("#product_#{product.id}") do
     click_on element
   end
+end
+
+Given(/^I am logged in as "([^"]*)"$/) do |name|
```
Comment (faraznaeem) [inline features/step_definitions/basic_steps.rb]: why are you using regex here?

File: features/user_can_review_order.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      |  email          | password  | name  |
+      |  john@doe.com   | password1 | John  |
+    When I visit the site
+    And I click on "Add to cart" on "pizza"
+    And I should see "1 item"
+    And I click on "Add to cart" on "kebab"
+    And I should see "2 items"
+      
+  Scenario: Visitor can review his order
+    Given I am logged in as "John"
```
Comment (faraznaeem) [inline features/user_can_review_order.feature]: move to background

File: features/user_can_review_order.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    And I click the "Proceed to checkout" button
```
Comment (faraznaeem) [inline features/user_can_review_order.feature]: `When`

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (pedbr) [reply features/step_definitions/basic_steps.rb]: Corrected!

File: features/user_can_review_order.feature (hunk 2 above)
Comment (pedbr) [reply features/user_can_review_order.feature]: Done!

File: features/user_can_review_order.feature (hunk 3 above)
Comment (pedbr) [reply features/user_can_review_order.feature]: Corrected!

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**